cmd/corectl/main.go

```diff
@@ -120,12 +133,15 @@ func configGenerator(db *sql.DB, args []string) {
 
 	conf := &config.Config{
 		IsGenerator: true,
-		IsSigner:    *isSigner,
 		Quorum:      quorum,
 		Signers:     signers,
 		MaxIssuanceWindow: chainjson.Duration{
 			Duration: *maxIssuanceWindow,
 		},
+		IsSigner:            *flagK != "",
```

Comment:
I think so. The idea here was to standardize the format between config.Nongenerator and config.Generator